src/pages/issues/index.tsx

```diff
@@ -82,6 +114,13 @@ const Issues: FC = () => {
         }
     };
 
+    useEffect(() => {
+        const searchString = getQueryStringFromUrl(router) as string;
+        if (router?.query?.dev === 'true' && searchString) {
+            fetchIssues(searchString);
+        }
+    }, [router?.query?.dev, searchString]);
```

Comment:
You are using `router?.query?.dev` multiple times, should be a variable.

Reply:
done